### PR TITLE
ssi_all: do not remove event.original in main ingest pipeline

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

See title.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [ ] 

## Related issues

- For #12067

### CODE REVIEW

**Review by chrisberkhout — CHANGES_REQUESTED** [review]
These changes look good.

Why was `okta` (owned by SSI) backed out?

Looking for `preserve_original_event` in SSI integrations I found...

Some were missed: `gcp` (because co-owned?) and `google_workspace` (owned by SSI).

Other relevant changes would be to remove the conditions in these:
- https://github.com/elastic/integrations/blob/e40fac2bacf9ec749ad5f76f5c75242cf661f90b/packages/falco/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml#L7-L11
- https://github.com/elastic/integrations/blob/e40fac2bacf9ec749ad5f76f5c75242cf661f90b/packages/qualys_vmdr/data_stream/knowledge_base/elasticsearch/ingest_pipeline/default.yml#L772-L777
- https://github.com/elastic/integrations/blob/e40fac2bacf9ec749ad5f76f5c75242cf661f90b/packages/qualys_vmdr/data_stream/asset_host_detection/elasticsearch/ingest_pipeline/default.yml#L1388-L1393

Also, off-topic, but the condition is wrong here:
- https://github.com/elastic/integrations/blob/e40fac2bacf9ec749ad5f76f5c75242cf661f90b/packages/mimecast/data_stream/audit_events/elasticsearch/ingest_pipeline/default.yml#L347-L351

File: packages/forgerock/data_stream/am_access/_dev/test/pipeline/test-am-access.log-expected.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
             "event": {
                 "action": "AM-ACCESS-ATTEMPT",
                 "id": "45463f84-ff1b-499f-aa84-8d4bd93150de-256203",
+                "original": "{\"payload\":{\"_id\":\"45463f84-ff1b-499f-aa84-8d4bd93150de-256203\",\"client\":{\"ip\":\"1.128.0.0\"},\"component\":\"OAuth\",\"eventName\":\"AM-ACCESS-ATTEMPT\",\"http\":{\"request\":{\"headers\":{\"accept\":[\"text/plain,*/*\"],\"content-type\":[\"application/x-www-form-urlencoded\"],\"host\":[\"openam-chico-poc.forgeblocks.com\"],\"user-agent\":[\"Jersey/2.34 (HttpUrlConnection 11.0.9)\"],\"x-forwarded-for\":[\"34.94.38.177, 34.149.144.150, 10.168.0.8\"],\"x-forwarded-proto\":[\"https\"]},\"method\":\"POST\",\"path\":\"https://openam-chico-poc.forgeblocks.com/am/oauth2/access_token\",\"secure\":true}},\"level\":\"INFO\",\"realm\":\"/\",\"request\":{\"detail\":{\"grant_type\":\"client_credentials\",\"scope\":\"fr:idm:*\"}},\"source\":\"audit\",\"timestamp\":\"2022-10-05T18:21:48.248Z\",\"topic\":\"access\",\"transactionId\":\"1664994108247-9f138d8fc9f59d23164c-26466/0\"},\"timestamp\":\"2022-10-05T18:21:48.249782137Z\",\"type\":\"application/json\"}",
```
[the remark this answers is not in the document]
Comment (efd6) [reply packages/forgerock/data_stream/am_access/_dev/test/pipeline/test-am-access.log-expected.json:14]: Yes, this is expected.

**Comment by efd6** [comment]
On deletion of the remove processor, elastic-package complains that the following remove processor requires an `if` attribute. This is incorrect, so I have omitted that change until the issue there is resolved.

google_workspace is omitted due to unrelated build failures.

**Comment by efd6** [comment]
For the mimecast issue noted in the PR comment. I don't know exactly what is going on there. I suspect that this is actually intended to be a remove of the `event.original` field, not the `source.ip` field given its location and the `source.ip` is never set.

I'll deal with that in a separate PR.

**Review by chrisberkhout — APPROVED** [review]

**Pull request merged**